<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeatStructure</class>
 <widget class="QWidget" name="NeatStructure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="FileGroupBox">
       <property name="title">
        <string>File management</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="FileHorizontalLayout">
          <item>
           <widget class="QPushButton" name="SavePushButton">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="LoadPushButton">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="AutomaticConfGroupBox">
       <property name="title">
        <string>Manual configuration</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="AutomaticConfHorizontalLayout">
          <item>
           <widget class="QPushButton" name="NewNodePushButton">
            <property name="text">
             <string>New node</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="NewConnectionPushButton">
            <property name="text">
             <string>New connection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DeletePushButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="CPPNCheckBox">
            <property name="text">
             <string>CPPN</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="RecurrentConnectionsCheckBox">
            <property name="text">
             <string>Recurrent Connections</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="ManualConfGroupBox">
       <property name="title">
        <string>Automatic configuration</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="ManualConfHorizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>In</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="InSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Out</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="OutSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="GeneratePushButton">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1094</width>
        <height>404</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="0">
        <widget class="QGroupBox" name="NEATConnectionGroupBox">
         <property name="title">
          <string>Connections</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="NEATConnectionVerticalLayout"/>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="NEATLayerGroupBox">
         <property name="title">
          <string>Layers</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="NEATLayerVerticalLayout"/>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="NEATNodeGroupBox">
         <property name="title">
          <string>Nodes</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="NEATNodeVerticalLayout"/>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
